src/Common/Crypto/CryptoGen.php

```diff
@@ -75,14 +93,14 @@ public function encryptStandard($value, $customPassword = null, $keySource = 'dr
     /**
      * Standard function to decrypt
      *
-     * @param  string  $value           This is the data to decrypt.
-     * @param  string  $customPassword  If provide a password, then will derive keys from this.(and will not use the standard keys)
-     * @param  string  $keySource       This is the source of the standard keys. Options are 'drive' and 'database'
-     * @param  int     $minimumVersion  This is the minimum encryption version supported (useful if accepting encrypted data
-     *                                   from outside OpenEMR to ensure bad actor is not trying to use an older version).
+     * @param string      $value          This is the data to decrypt.
+     * @param string|null $customPassword If provide a password, then will derive keys from this.(and will not use the standard keys)
+     * @param string      $keySource      This is the source of the standard keys. Options are 'drive' and 'database'
+     * @param int|null    $minimumVersion This is the minimum encryption version supported (useful if accepting encrypted data
+     *                                    from outside OpenEMR to ensure bad actor is not trying to use an older version).
      *
      */
-    public function decryptStandard($value, $customPassword = null, $keySource = 'drive', $minimumVersion = null)
+    public function decryptStandard(string $value, string $customPassword = null, string $keySource = 'drive', mixed $minimumVersion = null): false|string
```

Comment:
change $customPassword to `?string`
change $minimumVersion to `?int`
may make sense to change $value to `?string` also since guessing null values may get in there sometimes which would return empty string i think